### PR TITLE
Testing environment build & deploy

### PR DESCRIPTION
**Pull request by ivarpruijn**
Adds the ability to build & deploy to a test environment (testing.objectiv.io).

Todo:
- [x] Make deploy script work  (currently errors on `/usr/local/bin/docker-entrypoint.sh: 11: exec: deploytesting.sh: not found`)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,6 +27,10 @@ build-docker-build-image:
 	$(info building for ${OBJECTIV_ENVIRONMENT})
 	docker build --build-arg OBJECTIV_ENVIRONMENT=$(OBJECTIV_ENVIRONMENT) --no-cache -t objectiv/website-build:${TAG} -f docker/build/Dockerfile .
 
+build-docker-test-image: OBJECTIV_ENVIRONMENT="testing"
```
Comment (borft) [inline Makefile:30]: can't we simply do:
```bash
make OBJECTIV_ENVIRONMENT=testing build-docker-build-image
```?

File: .htaccess-testing [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# redirect to https
+RewriteEngine On
+RewriteCond %{HTTP_HOST} ^objectiv.io [NC]
+RewriteCond %{HTTP:X-Forwarded-Proto} !https
+RewriteCond %{HTTPS} off
+RewriteRule ^(.*)$ https://objectiv.io/$1 [R,L]
```
Comment (borft) [inline .htaccess-testing:6]: don't think this will do a lot

Comment (ivarpruijn) [reply .htaccess-testing:6]: As discussed, will leave it here for now, we can update this in the future if needed for e.g. adding a .htpasswd

File: Makefile (hunk 1 above)
Comment (ivarpruijn) [reply Makefile:30]: That should work, will update

**Pull request merged**